controllers/storagecluster/storageclasses.go

```diff
@@ -283,6 +283,10 @@ func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) S
 		disable:           managementSpec.DisableStorageClass,
 		isClusterExternal: initData.Spec.ExternalStorage.Enable,
 	}
+	if initData.Spec.ManagedResources.CephBlockPools.DefaultSCForCluster {
+		scc.storageClass.Annotations["storageclass.kubernetes.io/is-default-class"] = "true"
```

Comment:
use existing Const from Kubernetes package https://github.com/kubernetes/kubernetes/blob/dc8b57d8a75764e649d641ac77c1539a574d9aed/pkg/volume/util/storageclass.go#L32 ?

Reply:
define a const for it and use it as you are using it in 2 places.